v3/zone_test.go

```diff
@@ -27,602 +29,32 @@ var _ = Describe("Encode", func() {
-	{82.85338229176081, -37.96875, 14, "bD32858648748733"},
-	{81.5182718765338, 175.078125, 14, "aA18678460283758"},
-	{81.30832090051811, -177.1875, 14, "TK35253726505866"},
-	{84.11102985143594, -175.54571035156232, 14, "TO35083224064751"},
-	{84.52840283662707, 177.42303964843762, 14, "aE45177603556405"},
-	{85.05012909407333, 88.71932029724121, 14, "bb33563401287846"},
-	{85.0511287798066, 88.7024974822998, 14, "bb33563422636071"},
-	{-85.05491835052943, -80.244140625, 15, "BV801644720107680"},
-	{-85.05112877980659, -80.244140625, 15, "BV801672224571144"},
-	{-85.04354094565655, -76.81640625, 15, "BV752066087034602"},
-	{-84.92832092949963, 113.203125, 15, "CI584051202811030"},
-	{-85.05112877980659, 178.70361328125, 15, "KE053702425073344"},
-	{-85.05101030905541, 178.6212158203125, 15, "KE053474671871414"},
-	{-85.05112877980659, -179.47265625, 15, "DO048672464801367"},
-	{-85.04923290826916, -179.49462890625, 15, "DO048680771383421"},
-	{-78.63000556774834, -16.875, 15, "CZ312460160330640"},
-	{-76.9999351181161, 178.59375, 15, "LK482446631807432"},
-	{-77.69287033641926, -179.6484375, 15, "EU443120677000135"},
-	{-48.224672649565186, 61.171875, 15, "OK486263022716030"},
-	{-47.75409797968002, -101.6015625, 15, "Fb563137601173436"},
-	{-24.5271348225978, -160.3125, 15, "GH405610617770460"},
-	{-10.627811586244796, 175.58411714782733, 15, "Mc428447007470857"},
-	{-8.895519360279467, -174.9236953521727, 15, "GI714676472576137"},
-	{9.655313611088724, -177.7361953521727, 15, "QU430846555413857"},
-	{13.453737213419247, 177.1875, 15, "XK455868523435074"},
-	{28.613459424004414, 144.4921875, 15, "XM314886135481601"},
-	{22.268764039073965, 129.0234375, 15, "PS624076167125758"},
-	{30.401661513177608, 155.41321871032733, 15, "XL828827023882162"},
-	{42.77151327199906, 142.75696871032733, 15, "XP252183615213788"},
-	{-0.3515602939922709, -0.3515625, 15, "OY441655876848342"},
-	{0.6485910937534255, -2.438343789672672, 15, "OY450583438341310"},
-	{1.4061088354351594, -59.4140625, 15, "PA007023204648554"},
-	{27.010563124730247, -115.28990628967267, 15, "OI754343508378851"},
-	{47.989921667414166, -108.6328125, 15, "RX153284780083552"},
-	{44.33956524809713, 32.6953125, 15, "PZ541437131806627"},
-	{65.07213008560697, -80.859375, 15, "SV081012647253500"},
-	{73.82482034613932, 116.71875, 15, "ZA162768706170274"},
-	{78.27820145542813, 16.171875, 15, "aX538661552510561"},
-	{83.40004205976699, -38.671875, 15, "bD475537206875044"},
-	{82.30889251821553, 176.1328125, 15, "aA573170435224014"},
-	{82.07002819448266, -177.890625, 15, "TK720137660817775"},
-	{83.8992664793694, -177.8680312896727, 15, "TO078855632751174"},
-	{84.50259500130507, -179.9774062896727, 15, "TO448407012467243"},
-	{84.8609006021274, 178.96790621032733, 15, "aE801157747467437"},
-	{85.05112507763846, 89.37952995300293, 15, "bb337184418811744"},
-}
+var _ = Describe("Decode Position from Code", func() {
+
+	for _, tc := range loadCode2PositionTestCases() {
+		tc := &tc
```

Comment:
so this doesn't really work. try to `fmt.Println(tc.code)` inside the `It` block, you will see that it only ever prints the last one :(